src/boot_renesas.c

```diff
@@ -66,6 +66,10 @@ void do_boot(const uint32_t *app_offset)
     */
      app_entry = (void(*)(void))(*VECTOR_Reset_Handler);
      (*app_entry)();
+#elif defined(_RENESAS_RZN_)
+     app_entry = (void(*))(0x10010000);
```

Comment:
Please avoid hard coding. This should come from .config or a linker script.

Reply:
Thanks. I use an address value of linker script.